global_paths/plot_global_path.py

```diff
@@ -0,0 +1,99 @@
+import argparse
+import csv
+from typing import Dict, List, Tuple
+
+import plotly.graph_objects as go
+from custom_interfaces.msg import HelperLatLon
+
+from local_pathfinding.node_mock_global_path import MOCK_GPS, MockGlobalPath
+
+
+def main():
+    parser = argparse.ArgumentParser()
```

Comment:
Add an optional argument for position, so that we can generate a path from anywhere? Defaults to MOCK_GPS